examples/lib/stories/camera_and_viewport/camera_follow_and_world_bounds.dart

```diff
@@ -9,7 +9,7 @@ import 'package:flutter/services.dart';
 
 class CameraFollowAndWorldBoundsExample extends FlameGame
     with HasKeyboardHandlerComponents {
-  static const description = ''' 
+  static const description = '''
```

Comment:
weird that I didn't touch this file 🤔

Reply:
Strange, but let it slip with this one anyways, it's such a minor thing :smile:

Reply:
gotta get that PR count up spydon :P